ios/NativeSigner/Cards/HistoryCardExtended.swift

```diff
@@ -50,6 +50,14 @@ struct HistoryCardExtended: View {
                 line2: value.seedName + value.path + " in network with hash " +
                     value.networkGenesisHash.formattedAsString
             )
+        case let .secretWasExported(value):
+            HistoryCardTemplate(
+                image: .init(.eye, variants: [.trianglebadge, .exclamationmark, .fill]),
+                danger: true,
+                line1: "Secret was exported",
```

Comment:
After including changes on `master`. please align it by using localizable.

1. Open `Localizable.strings`, add there `"HistoryCard.SecretWasExported" = "Secret was exported";`
2. Build the app to regenerate the `enum Localizable`
3. replace this line with Suggested code


```suggestion
                line1: "Localizable.HistoryCard.secretWasExported.string
```

Reply:
No blocker for merging, I'm happy to refactor that on any of my next PRs 🙏🏻

Reply:
Done